SPECS/prometheus/prometheus.spec

```diff
@@ -0,0 +1,248 @@
+# Generated by go2rpm
+%ifnarch s390x
+%bcond_without check
+%endif
+
+%global builddate $(date +"%%Y%%m%%d-%%T")
+%global commit_version "e4487274853c587717006eeda8804e597d120340"
+
+Name:           prometheus
+Version:        2.24.1
+Release:        7%{?dist}
+Summary:        Prometheus monitoring system and time series database
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+License:        ASL 2.0
+URL:            https://github.com/prometheus/prometheus
+#Source0:        https://github.com/prometheus/prometheus/archive/refs/tags/v%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+# unzip Source0
+# run 'make assets' in it
+# rm -rf web/ui/react-app
+# tar czvf web-ui-2.24.1.tar.gz web/ui
+Source10:       web-ui-%{version}.tar.gz
+Source1:        prometheus.service
+Source2:        prometheus
```

Comment:
Nit: Can we use a more descriptive name for this file in our repo (e.g. `prometheus.sysconfig`)? You can install it in the buildroot with the current name, but it would be nice to be able to tell from the name what it does.